File: backend/src/main/java/by/fpmibsu/findafriend/controller/commands/places/UpdatePlaceCommand.java
package by.fpmibsu.findafriend.controller.commands.places;

import by.fpmibsu.findafriend.application.mediatr.Request;

public class UpdatePlaceCommand extends Request<Boolean> {
    public int placeId;
    public String country;
    public String region;
    public String city;
    public String district;

    public UpdatePlaceCommand(int placeId, String country, String region, String city, String district) {
        this.placeId = placeId;
        this.country = country;
        this.region = region;
        this.city = city;
        this.district = district;
    }
}

File: BackendNET/Backend/Commands/Places/UpdatePlace/UpdatePlaceCommand.cs
using MediatR;

namespace Backend.Commands.Places.UpdatePlace;

public class UpdatePlaceCommand : IRequest<bool>
{
    public int PlaceId { get; set; }
    public string? Country { get; set; }
    public string? Region { get; set; }
    public string? City { get; set; }
    public string? District { get; set; }
}